x-pack/plugins/secops/public/components/load_more_table/index.tsx

```diff
@@ -149,7 +164,21 @@ export class LoadMoreTable<T> extends React.PureComponent<BasicTableProps<T>, Ba
           </>
         )}
         <EuiTitle size="s">{title}</EuiTitle>
-        <EuiBasicTable items={pageOfItems} columns={columns} />
+        <EuiBasicTable
+          items={pageOfItems}
+          columns={columns}
+          onChange={onChange}
+          sorting={
+            sorting
+              ? {
+                  sort: {
+                    field: sorting!.field,
+                    direction: sorting!.direction === Direction.ascending ? 'asc' : 'desc',
+                  },
+                }
+              : {}
```

Comment:
If sorting isn't provided and we detect that we use an empty object of `{}`. Does this mean that `sorting` really wants to be an `{}` and not `null` as its default? Then you don't have to do any ternary operation and your code becomes more streamlined.